packages/in_app_purchase/ios/Classes/FIAPRequestHandler.m

```diff
@@ -10,13 +10,13 @@
 @interface FIAPRequestHandler () <SKProductsRequestDelegate>
 
 @property(copy, nonatomic) ProductRequestCompletion completion;
-@property(strong, nonatomic) SKProductsRequest *request;
+@property(strong, nonatomic) SKRequest *request;
```

Comment:
We need to change SKProductsRequest to SKRequest here because we need to handle SKRequests other than SKProductsRequest now. (SKRefreshReceiptRequest)